modules/network-observability-SRIOV-configuration.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// network_observability/configuring-operator.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-SR-IOV-config_{context}"]
+= Configuring the monitoring of SR-IOV interface traffic
+In order to collect traffic from a cluster with Single Root I/O Virtualization (SR-IOV) device, the `FlowCollector` `spec.agent.ebpf.privileged` specification value must be `true`. Then, the eBPF agent monitors other network namespaces in addition to the host network namespaces, which are monitored by default. When a pod with a virtual functions (VF) interface is created, a new network namespace is created. With `SRIOVNetwork` policy `IPAM` configurations specified, the VF interface is migrated from the host network namespace to the pod network namespace.
+
```

Comment:
> the VF interface is migrated from the host network namespace to the pod network namespace.

is this technical detail required in docs?

Reply:
@memodi I think @msherif1234 was wanting to make really clear that the 'SRIOVNetwork` spec is crucial to the migration. I'll let him weigh in and see if maybe we should tone down the technical details here.

Reply:
Got it, I understand it better now, we can that info ON. Resolving this.